### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by YZ775**

### CODE REVIEW

File: tools/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 cke-tools related changes.
 
+## 1.30.0 - 2024-08-23
+
+- Update Go modules dependencies [#766](https://github.com/cybozu-go/cke/pull/765)
```
Comment (chez-shanpu) [inline tools/CHANGELOG.md:7]:
```suggestion
- Update Go modules dependencies [#766](https://github.com/cybozu-go/cke/pull/766)
```

#765 seems to be closed.

File: tools/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 cke-tools related changes.
 
+## 1.30.0 - 2024-08-23
```
Comment (chez-shanpu) [inline tools/CHANGELOG.md:5]:
```suggestion
## 1.30.0 - 2024-09-18
```

The date looks old.

**Review by chez-shanpu — COMMENTED** [review]
Nice work :)
I left a few small comments.

File: tools/CHANGELOG.md (hunk 2 above)
Comment (YZ775) [reply tools/CHANGELOG.md]: Thank you! I fixed.

**Review by chez-shanpu — APPROVED** [review]
LGTM

**Pull request merged**